### ISSUE TITLE
Will you publish weights?

### ISSUE THREAD

**Issue by Sid1057** [zhyever/LiteDepth#1]
Hello!
First of all thanks for a great work!

Will you publish weights later?

**Comment by zhyever** [comment]
Yes. But that can be much later, since I'm looking for a PhD position.

**Comment by Sid1057** [comment]
Any updates sir?🙂

**Comment by vikrantshinde2003** [comment]
> Yes. But that can be much later, since I'm looking for a PhD position.
Sir when will you upload the weights?

**Comment by vikrantshinde2003** [comment]
> Any updates sir?🙂

Did you get any further updates on this?
